src/psyclone/tests/generator_test.py

```diff
@@ -544,8 +545,8 @@ def test_main_invalid_api(capsys):
     assert str(excinfo.value) == "1"
     _, output = capsys.readouterr()
     expected_output = ("Unsupported API 'madeup' specified. Supported API's "
```

Comment:
This should really be "unsupported APIs" instead of "unsupported API's" (plural, not possesive). I would kindly ask for the error message and the associated tests to be updated.

Reply:
Good s'pot. I've also updated generator.py which needed copyright update and I got rid of the executable line and encoding information at the top of the file.